sig-scalability/charter.md

```diff
@@ -0,0 +1,110 @@
+# SIG Scalability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+SIG Scalability primary responsibilities are to define scalability goals for
+Kubernetes, define, test and measure performance and scalability related
```

Comment:
s/,define,/

Reply:
Why? We actually define SLIs/SLOs (with cooperation with other sigs).

Reply:
Gramatically the sentence is weird.  

perhaps 
`SIG Scalability primary responsibilities are to define scalability goals for
Kubernetes.  This involves defining ...`

Reply:
Thanks for suggestion - done.